packages/oft-sand/.solhint.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "solhint:recommended",
+  "plugins": ["prettier"],
+  "rules": {
+    "prettier/prettier": [
+      "error",
+      {
+        "endOfLine": "auto"
+      }
+    ],
+    "compiler-version": ["error", "^0.8.0"],
```

Comment:
To get rid of the custom errors warning you can add `"custom-errors": "off",` for the time being.
Later, let's update solhint to latest version (5.0.1) and review

Reply:
Then, there are only 4 warnings remaining for you to fix in contracts/sand

Reply:
done